homeassistant/components/asuswrt/device_tracker.py

```diff
@@ -92,17 +94,19 @@ def mac_address(self) -> str:
         return self._device.mac
 
     @callback
-    def async_on_demand_update(self):
+    def async_on_demand_update(self) -> None:
         """Update state."""
         self._device = self._router.devices[self._device.mac]
         self._attr_extra_state_attributes = {}
         if self._device.last_activity:
             self._attr_extra_state_attributes[
                 "last_time_reachable"
             ] = self._device.last_activity.isoformat(timespec="seconds")
+        if self._device.is_connected:
+            self._attr_extra_state_attributes["connected_to"] = self._router.host
```

Comment:
This looks like a new feature. I'd expect it to be a separate PR.

Reply:
Forget about this, removing...